HDIClusterConfigurationManagement/src/main/java/com/microsoft/configuration/management/Main.java

```diff
@@ -53,8 +76,6 @@ public static void main(String[] args) throws  Exception {
         String ambariPassword = configuration.get("ambariPassword").getAsString();
         String clusterName = ambariUrl.getHost().split("\\.")[0];
 
-        SkipServerCertificateValidation();
```

Comment:
Can you add an optional argument to SkipServerCertificateValidation? The default will be false but we should also be able to set it to true. Otherwise we won't be able to run the tool with a test cluster.